price_pusher/src/index.ts

```diff
@@ -11,13 +11,21 @@ import { EvmPriceListener, EvmPricePusher, PythContractFactory } from "./evm";
 import { PythPriceListener } from "./pyth-price-listener";
 import fs from "fs";
 import { readPriceConfigFile } from "./price-config";
+import { PriceServiceConnection } from "@pythnetwork/pyth-common-js";
+import { InjectivePriceListener, InjectivePricePusher } from "./injective";
 
 const argv = yargs(hideBin(process.argv))
-  .option("evm-endpoint", {
+  .option("network", {
```

Comment:
Can you also update the readme with this new configs and values? You would need to update the compose files too.

Reply:
I am not sure if this file is final yet. 
I have added the code to it just so that injective pieces are working fine. 
I will refactor this file too and then will update the README

Reply:
command line options will be annoying as the configuration becomes more complicated. I suggest moving this stuff to a config file. I note that there's already a price config file, so maybe it goes in there.